packages/form-js-playground/src/components/PlaygroundRoot.js

```diff
@@ -137,6 +137,30 @@ export function PlaygroundRoot(props) {
     paletteRef.current = formEditor.get('palette');
     propertiesPanelRef.current = formEditor.get('propertiesPanel');
 
+    formEditor.on('formField.add', ({ formField }) => {
```

Comment:
What I'm now wondering after having a second look, does this work with `undo` / `redo`? I think undo, redo might anyway not work super sophisticated in the Form Playground, so it might be ok for now.

Reply:
This basically resets the edit history, I couldn't find a better API to properly append the history

Reply:
We have the concept of [modeling behaviors](https://github.com/bpmn-io/form-js/tree/develop/packages/form-js-editor/src/features/modeling/behavior) that intercept `commands` as removing, adding form fields, updating IDs etc. That ensures proper handling of the Form Editor's command stack and ensures we properly clean up things on undo/redo.

In this case, we use a simple event listener that runs _after_ the `formField.add` command, so it would not run on undo/redo, e.g. the JSON editor would not properly remove the demo data. I think that's not a problem currently, but something you could think about in the future to implement this as a proper modeling behavior.

Reply:
cc @Skaiir

Reply:
ok, merging this for now then

Reply:
@vsgoulart 

What's the use case for the demo data? To show something in the editor?

Reply:
> @vsgoulart
> 
> What's the use case for the demo data? To show something in the editor?

@Skaiir yes, the goal is to make it easier to get "onboarded" on this new component so he/she doesn't just see an empty table in the preview